src/pyscript/plugins/create.py

```diff
@@ -1,15 +1,61 @@
+import time
+import webbrowser
+from pathlib import Path
+from typing import Optional
+
 import typer
 
-from pyscript import LATEST_PYSCRIPT_VERSION, app, cli, plugins
-from pyscript._generator import create_project
+from pyscript import LATEST_PYSCRIPT_VERSION, app, cli, console, plugins
+from pyscript._generator import create_project, file_to_html, string_to_html
+
+
+def wrap_file(title, output, command, show, app_or_file_name, pyscript_version):
+    title = title or "PyScript App"
```

Comment:
Docstring

Reply:
Removed this altogether because of the new implementation.